packages/core/ios/Sources/DdSdk.mm

```diff
@@ -14,10 +14,15 @@
 
 @implementation DdSdk
 
-static __weak RCTBridge *s_latestBridge = nil;
+static __weak RCTBridge *_bridge = nil;
+
+- (void)setBridge:(RCTBridge *)bridge {
+    [super setBridge:bridge];
```

Comment:
By being explicit about this we ensure that this setter does not get removed nor lost again.